modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// oadp-features-plugins-known-issues
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-features-plugins-known-issues_{context}"]
+= OADP plugins known issues
+
+The following section describes known issues in {oadp-first} plugins:
+
+[id="velero-plugin-panic_{context}"]
+== Velero plugin panics on imagestream backup due to a missing secret
+
+The velero plugin panics on imagestream backup. A missing secret causes this panic error.
+
+When the backup and the Backup Storage Location (BSL) are managed outside the scope of the Data Protection Application (DPA), the OADP controller, meaning the DPA reconciliation does not create the relevant `oadp-<BSL Name>-<BSL Provider>-registry-secret`.
```

Comment:
I am not clear on what this means:   _meaning the DPA reconciliation_

Reply:
You reconcile a DPA by running a command such as:

```
oc get dpa dpa-sample -n openshift-adp -o jsonpath='{.status}'
```

Example output
```
{"conditions":[{"lastTransitionTime":"2023-10-27T01:23:57Z","message":"Reconcile complete","reason":"Complete","status":"True","type":"Reconciled"}]}
```